authors/apps/profiles/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+from ..authentication.models import User
+from django.db.models.signals import post_save
+
+
+# Sample request data
```

Comment:
@Siffersari remove all the unnecessary comments

Reply:
Thank you for that. I have removed the comments